src/views/Pools/components/CakeVaultCard/FeeSummary.tsx

```diff
@@ -15,11 +15,28 @@ const FeeSummary: React.FC<FeeSummaryProps> = ({ stakingTokenSymbol, lastDeposit
   const TranslateString = useI18n()
   const feeAsDecimal = parseInt(vaultFees.withdrawalFee) / 100
   const feeInCake = (parseFloat(stakeAmount) * (feeAsDecimal / 100)).toFixed(4)
+  const { targetRef, tooltip, tooltipVisible } = useTooltip(
+    <Box>
+      <Box style={{ fontWeight: 'bold' }} mb="4px">
+        {`${TranslateString(999, `Unstaking fee:`)} ${feeAsDecimal}%`}
+      </Box>
```

Comment:
Why not use `<Text />` here?

Reply:
I guess its because theme for tooltip is inverted but the inversion doesn't happen to children automatically.
But you can use `<Text />` just need to specify the color of the opposite theme

Reply:
The colors... 

I couldn't get the Tooltip's text colors to be inherited by the Text component, they unset it to the Text default, and then I don't think any other colors match the dark/light mode colors required by the tooltip?

<img width="330" alt="Screenshot 2021-04-30 at 07 56 19" src="https://user-images.githubusercontent.com/79279477/116659475-91671e00-a989-11eb-83b2-d947f272538b.png">

Reply:
yeah I'd say fuck it, lets use the box for now, seems to work. I will take a look how to auto-invert the children theme and will refactor it
People already asking questions and need tooltips ASAP I guess.